commands/sha1/index.js

```diff
@@ -0,0 +1,11 @@
+sha1 = require('js-sha1');
```

Comment:
As you are requiring the js-sha1 package you have to add it to the _package.json_

You can also use the -s or --save flag when you perform an npm install: _npm install js-sha1 -s_
Running the command with this flag automatically updates the _package.json_

Reply:
Ok, I'll do that